src/citra_qt/configuration/config.cpp

```diff
@@ -872,29 +871,13 @@ void QtConfig::ReadUILayoutValues() {
     qt_config->endGroup();
 }
 
-void QtConfig::ReadWebServiceValues() {
-    qt_config->beginGroup(QStringLiteral("WebService"));
-
-    NetSettings::values.web_api_url =
-        ReadSetting(QStringLiteral("web_api_url"), QStringLiteral("https://api.citra-emu.org"))
```

Comment:
None of the web service settings should be removed, there are currently third party services that allow using multiplayer rooms, and those require changing these settings manually.